src/components/input/GeoInput/index.js

```diff
@@ -202,6 +216,10 @@ export default class GeoInput extends React.PureComponent {
         const polygons = this.getPolygons(value);
         const options = this.getAllGeoOptions(geoOptionsByRegion);
 
+        const polygonSupportIndex = accessibleFeatures.findIndex(f => f.key === 'polygon_support_geo');
+        const isPolygonFeatureEnabled = polygonSupportIndex !== -1;
+        const shouldEnablePloygon = isPolygonFeatureEnabled && polygonsEnabled;
```

Comment:
Spelling mistake for "Polygon"
Try using this plugin for vim https://github.com/kamykn/spelunker.vim